src/ui/AccountStatus.svelte

```diff
@@ -1,21 +1,57 @@
 <script>
-  import Svg from '@/ui/Svg/svelte'
-  import { getSanbaseSubscription, getTrialDaysLeft } from '@/utils/subscription'
+  import { customerData$ } from '@/stores/user'
+  import { subscription$ } from '@/stores/subscription'
+  import { PlanName } from '@/utils/plans'
+  import { AccountDropdown } from '@/utils/enums'
 
   export let currentUser
+  export let variant = AccountDropdown.Version1
 
-  $: subscription = currentUser && getSanbaseSubscription(currentUser.subscriptions)
-  $: trialDaysLeft = subscription && getTrialDaysLeft(subscription)
-  $: trialPostfix = trialDaysLeft ? ` Trial (${trialDaysLeft})` : ''
+  $: ({ isEligibleForTrial, annualDiscount } = $customerData$)
+  $: trialSpecialOfferPercent =
+    (annualDiscount &&
+      annualDiscount.isEligible &&
+      annualDiscount.discount &&
+      annualDiscount.discount.percentOff) ||
+    0
+  $: subscription = $subscription$
+  $: currentPlanName = subscription && (PlanName[subscription.plan.name] || subscription.plan.name)
 </script>
 
 {#if currentUser}
-  {#if subscription}
-    <a
-      href="https://academy.santiment.net/products-and-plans/sanbase-pro-features/"
-      class="pro btn-1 btn--s caption row v-center">
-      <Svg id="crown" w="13" h="10" class="mrg-s mrg--r" />
-      Pro{trialPostfix}</a>
+  {#if currentPlanName && currentPlanName.toUpperCase().includes('PRO')}
+    <div class="row v-center">
+      {#if trialSpecialOfferPercent > 0}
+        <a
+          href="https://app.santiment.net/pricing"
+          class="btn-1 btn-2 btn--orange mrg--r mrg-m"
```

Comment:
`btn-2` has a higher priority, therefore it's better to place it first. Same for `mrg-m`
```suggestion
          class="btn-2 btn-1 btn--orange mrg-m mrg--r"
```